src/models/AppState.js

```diff
@@ -91,11 +99,11 @@ function (Okta, Q, Factor, BrowserFeatures, Errors) {
       if (this.settings.get('features.securityImage')) {
         var self = this;
         this.listenTo(this, 'change:username', function (model, username) {
-          getSecurityImage(this.get('baseUrl'), username)
+          getSecurityImage(this.get('baseUrl'), username, model.get('deviceFingerprint'))
```

Comment:
Where does deviceFingerprint get set on the model?

Reply:
it is set in ```PrimaryAuthController```, on a few lines that look like ```options.appState.set('username', self.model.get('username'))```.
The model here is the same as ```this```: it is the AppState. So you bring a good point, I should stick to one or the other, so I'll change it to ```this.get('deviceFingerprint')```